test/public/pixelmatch.js

```diff
@@ -0,0 +1,157 @@
+'use strict'
+
+function pixelmatch (img1, img2, output, width, height, options) {
```

Comment:
Is this [mapbox's](https://github.com/mapbox/pixelmatch/blob/master/index.js)? If so, maybe add it as a devDependency instead (for attribution/licensing and less code for us to maintain)?

Reply:
is called pixelmatch, the reason i copy pasted it is unclear now. I think i did not  manage to use on the webserver as a npm module? i ll try again